### PR TITLE
po.pl - Polish translation added

### PR DESCRIPTION
**Pull request by DanyGee**
Polish translation added.

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -21,363 +21,363 @@ msgstr ""
...
 #. columns
 #: src/view/TransactionView.vala:538
 msgid "Date"
-msgstr ""
+msgstr "Data"
 
 #: src/view/TransactionView.vala:551
 msgid "Merchant"
-msgstr ""
+msgstr "Klient"
```
Comment (cjfloss) [inline po/pl.po]: Using Google Translate i get Klient translated to customer.
In this case "Merchant" is the one selling the product.
I think using Klient might cause some confusion, but I don't speak Polish. 

Is this the correct term?

File: po/pl.po [hunk 2]
```diff
@@ -21,363 +21,363 @@ msgstr ""
...
 
 #: src/view/AccountWelcomeScreen.vala:41
 msgid "Spend! Get paid!"
-msgstr ""
+msgstr "Wydawaj! Zarabiaj!"
 
 #: src/view/AccountWelcomeScreen.vala:41
 msgid "There are currently no transactions in this account"
-msgstr ""
+msgstr "Aktualnie braj transakcji dla tego konta"
```
Comment (cjfloss) [inline po/pl.po]: Google Translate suggests using  "Aktualnie **brak** transakcji dla tego konta"

File: po/pl.po (hunk 1 above)
Comment (DanyGee) [reply po/pl.po]: "Merchant" in polish means "Kupiec", someone who mainly buys stuff and occasionally sells it.
I wanted to propose something more general, like "client", "subject", "entity", something that will fit most outcome and income possibilities. Would you call a merchant your employer from which you recieve your monthly payment?

File: po/pl.po (hunk 2 above)
Comment (DanyGee) [reply po/pl.po]: Yup, this was a typo, sorry.

**Comment by cjfloss** [comment]
Thanks for the translation!

I added a few comments you should look.

**Comment by DanyGee** [comment]
Added my answers. Please edit the file accordingly.

File: po/pl.po (hunk 1 above)
Comment (DanyGee) [reply po/pl.po]: I think the best for polish version will be "Podmiot" = "Entity". This way it doesnt matter if you spent or earned money.

Comment (cjfloss) [reply po/pl.po]: You are right, it should be something like "subject", "second party" or, as HomeBank uses, "Payee".

**Pull request merged**